ops/terraform/services/base/main.tf

```diff
@@ -1,22 +1,20 @@
 locals {
-  env              = terraform.workspace
-  is_ephemeral_env = !(contains(local.established_envs, local.env))
-
-  established_envs = [
-    "test",
-    "mgmt",
-    "prod-sbx",
-    "prod"
-  ]
 
   default_tags = {
-    Environment    = local.env
+    Environment    = local.seed_env
     application    = "bfd"
     business       = "oeda"
     stack          = local.env
     Terraform      = true
     tf_module_root = "ops/terraform/services/base"
   }
+
+  env              = terraform.workspace
+  established_envs = ["test", "prod-sbx", "prod"]
+  seed_env         = one([for x in local.established_envs : x if can(regex("${x}$$", local.env))])
+
+  is_ephemeral_env = local.env != local.seed_env
```

Comment:
Similar logic is found in each terraservice and is practically begging to be replaced with a re-usable child module.